journals/apps/journals/models.py

```diff
@@ -314,6 +315,17 @@ def generate_basket_url(self, sku):
         ))
         return basket_url
 
+    def update_related_objects(self, deactivate=False):
+        data = {
+            "status": "active" if not deactivate else "inactive",
+            "card_image_url": self.site.root_url + self.card_image.file.url,
+            "title": self.title,
+            "full_description": self.long_description,
+            "short_description": self.short_description,
+        }
+        discovery_journal_api_client = self.site.siteconfiguration.discovery_journal_api_client
+        discovery_journal_api_client.journals(self.journal.uuid).patch(data)
```

Comment:
So on initial run of publish_journal mgmt command this ends up being called because the about page is being published https://github.com/edx/journals/blob/master/journals/apps/journals/management/commands/publish_journals.py#L115 but it's not necessary to modify it here as we've just set it during creation and nothing has changed. 
Wonder if there is a way to only call the patch if the data is different, or somehow ignore the publish if it was done from the mgmt command?